src/main/features/response/draft/responseDraft.ts

```diff
@@ -2,6 +2,7 @@ import { Response } from 'response/form/models/response'
 import { Serializable } from 'models/serializable'
 import { FreeMediation } from 'response/form/models/freeMediation'
 import Defence from 'response/form/models/defence'
+import HowMuchIsPaid from 'features/response/form/models/howMuchPaid'
```

Comment:
HowMuchIsPaid shouldn't be default export.

It was decided in new code we don't use default exports as Intelij seems to have problems with them sometimes.